R/dm_get_tables.R

```diff
@@ -0,0 +1,31 @@
+#' Get tables
+#'
+#' `dm_get_tables()` returns a named list of \pkg{dplyr} [tbl] objects
```

Comment:
Should the print method be clearer regarding what's returned when it's `dm_keyed_tbl`?

Reply:
Are you referring to the `print()` method for keyed tables?

Reply:
yes! it says "a tibble" only